src/types.ts

```diff
@@ -101,6 +101,7 @@ export interface CucumberSuite {
     import?: string[];
     tags?: string[];
     format?: string[];
+    formatOptions?: object;
```

Comment:
Can we be more precise? Currently in saucectl, the format options field is a `map[string]string`. Both keys and values will come in their string form.